### PR TITLE
Release 2.0.1

### PR DESCRIPTION
**Pull request by PierreRambaud**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Description?  | Release 2.0.1.
| Type?         | improvement

### CODE REVIEW

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+GitHub contributors:
+--------------------------------
+ - Francois Gaillard
+ - Jerome Nadaud
```
Comment (atomiix) [inline CONTRIBUTORS.md:4]:
```suggestion
```

**Review by matks — DISMISSED** [review]

**Comment by florine2623** [comment]
Hello @PierreRambaud ,

I can't perform the upgrade from v2.0.0 to v2.0.1 on this PR, could you take a look ? :)
Screenshot 2022-01-05 at 16 27 28

Thanks!

File: upgrade/upgrade-2.0.1.php [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ */
+
+/**
+ * @param Module $module
+ *
+ * @return bool
+ */
+function upgrade_module_2_0_1$module)
```
Comment (PierreRambaud) [inline upgrade/upgrade-2.0.1.php:27]:
```suggestion
function upgrade_module_2_0_1($module)
```

**Comment by PierreRambaud** [comment]
@florine2623 Sorry again, should be better right now

**Comment by florine2623** [comment]
Thanks @PierreRambaud ,

Upgrade works well now :)

On PS 1.7.8.x : QA ✅ 

On develop branch : QA 🚫 
The Read more link is missing :
Screenshot 2022-01-10 at 14 19 45

I should have the following :
Screenshot 2022-01-10 at 14 19 58

Could you check ?
Thanks!

**Comment by kpodemski** [comment]
@florine2623 I think it's ok because "Read more" is related to the Addons Marketplace and has probably been removed.

**Comment by PierreRambaud** [comment]
> @florine2623 I think it's ok because "Read more" is related to the Addons Marketplace and has probably been removed.

Exactly, it doesn't concerns the module itself

**Comment by florine2623** [comment]
Ok perfect then !
It is QA ✅ 
Thanks all !

**Review by kpodemski — APPROVED** [review]

**Comment by kpodemski** [comment]
Thanks @florine2623 & @PierreRambaud

**Pull request merged**